### DISCUSSION TITLE
How to solve a case with select dropdown show incomplete in a scroll box ?

### DISCUSSION THREAD

**Discussion by kabeep** [ant-design/ant-design#38327] [Q&A] [answered]
### Steps to reproduce
There's always have a scene in my work with dropdown in a modal , also these modal often have a scroll bar.
Like this :
![image](https://user-images.githubusercontent.com/56389128/199179692-e01a6842-dba4-4f99-a9d9-d0e8176b2b72.png)

I have to use getPopupContainer attribute to prevent select dropdown scrolling with the page.
If i not :
![image](https://user-images.githubusercontent.com/56389128/199181742-53c3f301-c81c-4987-8612-8cd2929d414b.png)

Now we have another trouble with select dropdown display incomplete , because the content is hide when overflow.
List this :
![image](https://user-images.githubusercontent.com/56389128/199181351-ecb13270-5d03-446b-b262-d690b9094f88.png)

### What is expected?
I want to make them generic , so i can't modify the height of the container and dropdown.
How to solve the case with a neat way ?
sincere thanks ! 由衷感谢！

<html>
<body>

Environment | Info
-- | --
antd | 4.23.3
React | 17.0.0
System | Windows 10
Browser | Google Chrome

</body>
</html>

**Comment by zombieJ** [comment, marked as answer]
When you use `overflow: auto`, the content inside with absolute position will be hidden in container also. It's css logic which may not change.
Or keep the popup force align when trigger onScroll event instead
